pipelines/cdp/cdp_pipeline/generate_transaction_data.py

```diff
@@ -8,17 +8,16 @@
 
 async def publish_coupons_to_pubsub():
     bucket_name = '<bucket_name>'
-    project_id = '<project_id>'
+    project_id = 'malhotradi'
```

Comment:
Remove hard coded data from the script. Ideally, this should be populated from the script that sets all the environment variables (e.g. having the script read a variable from the environment)